web/about.html

```diff
@@ -0,0 +1,95 @@
+	let docEl = document.documentElement;
+	let width = Math.max(300,docEl.clientWidth);
+	let height = Math.max(300,docEl.clientHeight);
+	docEl.style.setProperty("--vw-unit",`${(width / 100).toFixed(1)}px`);
+	docEl.style.setProperty("--vh-unit",`${(height / 100).toFixed(1)}px`);
+}
+</script>
+</head>
+<body>
+
+<header>
+
+<h1 id="logo">YOU<i class="bold-period" aria-label="stylized period in red color"></i></h1>
+
+<nav>
+	<ul>
+		<li><a href="/">Home</a></li>
+		<li><a href="about.html">About</a></li>
+		<li><a href="privacy-policy.html">Privacy Policy</a></li>
+	</ul>
+</nav>
+
+</header>
+
+<main>
+
+<h2>A privacy-first period tracking app.</h2>
+
+<p>We could almost all agree that it's never been more important for us to OWN OUR INFORMATION, not governments or "big tech" companies, so that we decide what is done with that information.</p>
+
+<p>We believe that especially goes for private medical/health data, of which menstrual tracking (for women / people with uteruses) might just be some of the most sensitive and important to protect.</p>
+
+<p>Companies and apps can promise all they want that they're being safe and private with our information. But if they have our information, and they're a company, they're going to have to make money somehow, either by charging us for it, or worse, charging someone else to use it in some way.</p>
+
+<p>We believe this kind of private and sensitive medical information belongs to you, and you alone. As such, YOU. (the app) puts you in control of all your information, and never collects or tracks anything about you, not even your name or email address. Since we don't have any of your data, we obviously CANNOT sell it or hand it over to any governmental authority.</p>
+
+<p>Everything you enter into YOU. (the app) is yours, always and only yours. It stays on your device, protected and secure, and you decide what to do with that information.</p>
+
+<p>This app is free to use, and will remain so forever.</p>
+
+<p>Once installed, this web-app (PWA) runs entirely offline -- only locally on your device, with no need for any internet or to connect to any remote service -- and uses strong cryptography practices to keep your data secure and private on your device, ONLY.</p>
+
+<p>It's built with love and respect, for you, and only about you, and only to help you keep and track your own private health information. That's it. Period.</p>
+
+</main>
+
+<footer>&copy;2022 YOU<i class="bold-period" aria-label="stylized period in red color"></i></footer>
+
+<div id="notification-banner" class="hidden">
+	<form>
+		<div class="msg" aria-live="off" aria-atomic="true"></div>
+		<button type="button" class="cancel-btn" class="hidden">Cancel</button>
+		<button type="submit">OK</button>
+	</form>
+</div>
+
+
+<!----------------------------------->
+
+<script src="js/external/base64-arraybuffer.umd.js"></script>
+<script src="js/util.js"></script>
+<script type="module" src="js/main.js"></script>
```

Comment:
these 3 above scripts don't need to be on the about page (and indeed will cause some JS errors). the "window-resize" one below needs to be here tho.